WordPress/WordPressTest/Tenor/TenorTests.swift

```diff
@@ -27,4 +27,12 @@ class TenorTests: XCTestCase {
         XCTAssertNotNil(gif.url, "Gifs should have a url")
         XCTAssertNotNil(gif.preview, "Gifs should have a preview url")
     }
+
+    func testTonerResponsePaging() {
```

Comment:
We can rename this file to `TenorResponseTests` and create a new `TenorPageableTests` with this one.